src/components/motorcycleForm/addMotorcycle.js

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { createMotorcycle } from '../../redux/motorcycles/motorcycles';
+
+const AddMotorcycle = () => {
+  const getData = () => ({
+    model: '',
+    price: 0,
+    duration_months: 0,
+    description: '',
+    image: '',
+  });
+  const [formData, setFormData] = useState(getData);
+
+  const dispatch = useDispatch();
+
+  const currentUser = useSelector((state) => state.auth);
+
+  useEffect(() => {
+    localStorage.setItem('formData', JSON.stringify(formData));
+  }, [formData]);
```

Comment:
Could you specify what the above piece of code does?
You should remove it if it does not perform any function.